### PR TITLE
Release 4.4.6

### PR DESCRIPTION
**Pull request by fspmarshall**
Patch release for #5105

### CODE REVIEW

**Comment by awly** [comment]
Can we wait for https://github.com/gravitational/teleport/pull/5120 to be included?

**Comment by fspmarshall** [comment]
> Can we wait for #5120 to be included?

@awly Yeah, no problem.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+### 4.4.6
+
+This release of teleport contains a bug fix.
+
+* Fix an issue where `tsh login` would fail with an `AccessDenied` error if
+the user was perviously logged into a leaf cluster. [#5105](https://github.com/gravitational/teleport/pull/5105)
+
```
Comment (awly) [inline CHANGELOG.md:9]:
```suggestion

* Patch a SAML authentication bypass (see https://github.com/russellhaering/gosaml2/security/advisories/GHSA-xhqq-x44f-9fgg): [#5120](https://github.com/gravitational/teleport/pull/5120). Users with SAML SSO providers should upgrade as soon as possible.
```

**Review by awly — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+### 4.4.6
+
+This release of teleport contains a security fix and a bug fix.
+
+* Patch a SAML authentication bypass (see https://github.com/russellhaering/gosaml2/security/advisories/GHSA-xhqq-x44f-9fgg): [#5120](https://github.com/gravitational/teleport/pull/5120). Users with SAML SSO providers should upgrade as soon as possible.
```
Comment (webvictim) [inline CHANGELOG.md:7]: I think we should add similar wording to the last time we did one of these patch releases:

```suggestion
* Patch a SAML authentication bypass (see https://github.com/russellhaering/gosaml2/security/advisories/GHSA-xhqq-x44f-9fgg): [#5120](https://github.com/gravitational/teleport/pull/5120).

Any Enterprise SSO users using Okta, Active Directory, OneLogin or custom SAML connectors should upgrade their auth servers to version 4.4.6 and restart Teleport. If you are unable to upgrade immediately, we suggest disabling SAML connectors for all clusters until the updates can be applied.
```

**Review by webvictim — APPROVED** [review]

**Pull request merged**